spec/crinder_spec.cr

```diff
@@ -47,6 +47,18 @@ class NilableTodoRenderer < TodoRenderer
   remove updated
 end
 
+record TodoSubtasked, name : String, subtasks : Array(Subtask)
+record Subtask, name : String
```

Comment:
I prefer moving the these `record`s to L4, under `record Todo`.

Reply:
I think fixtures should be simple and focused to what's important.